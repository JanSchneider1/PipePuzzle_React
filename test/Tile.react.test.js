const React = require('react');
const { create } = require('react-test-renderer');

const { Tile } = require("../src/js/components/Tile.react");
const { TileData } = require("../src/js/_tiledata");

test('Creating Tile matches Snapshot', () => {
    const component = create(
        <Tile data={new TileData('-',0)}/>,
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});