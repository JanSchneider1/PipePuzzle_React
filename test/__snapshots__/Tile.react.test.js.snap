// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Creating Tile matches Snapshot 1`] = `
<div
  className="tile rotate-0"
>
  <img
    alt="Tile"
    onClick={[Function]}
    src="/img/Empty_Tile.png"
  />
</div>
`;
